script.js

```diff
@@ -149,19 +146,14 @@ $(document).ready(function(){
   }
 
   var active = function(object){
-    if(object.prop('id')!= null){
-      $(object).addClass("light");
-      oscillator.frequency.value = colorIndex[object.prop('id')]*100;
-      oscillator.connect(gainNode);
-    }
+    $(object).addClass("light");
```

Comment:
does `object` mean `DOM element` here? If so I'd name this variable `element` or `node`